driver-core/src/main/com/mongodb/ServerAddress.java

```diff
@@ -129,6 +129,16 @@ public ServerAddress(final String host, final int port) {
         this.port = portToUse;
     }
 
+    private String join(final String delimiter, final String[] array, final int start, final int end) {
```

Comment:
You're right.  Was using that for a failed approach and missed it in the clean up.